client/src/utils/uploadbox.js

```diff
@@ -245,13 +245,15 @@ export class UploadQueue {
                 }
             });
         }
+        // Returns last added file index.
+        return this.nextIndex - 1;
```

Comment:
We should not reduce this, the `id` is currently always a unique arbitrary number. Reducing this index might lead to duplicate model ids.